lib/overcommit/hook/pre_commit/case_conflicts.rb

```diff
@@ -0,0 +1,20 @@
+module Overcommit::Hook::PreCommit
+  # Checks for files that would conflict in case-insensitive filesystems
+  # Adapted from https://github.com/pre-commit/pre-commit-hooks
+  class CaseConflicts < Base
+    def run
+      paths = Set.new(applicable_files.map { |file| File.dirname(file) })
```

Comment:
Ach, my bad. I was adding the trailing slash without thinking when testing `git ls-tree`. Fix incoming.